python/oneflow/framework/docstr/dot.py

```diff
@@ -0,0 +1,55 @@
+"""
+Copyright 2020 The OneFlow Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import oneflow as flow
+from oneflow.python.nn.module import Module
+from oneflow.python.oneflow_export import oneflow_export, experimental_api
+
+
+@oneflow_export("dot")
+@experimental_api
+def dot(input, other):
+    r"""This operator computes the dot product of tensor input and other.
+    
+    The equation is:
+    
+	$$		
+	​   \\sum_{i=1}^{n}(x[i] * y[i])
```

Comment:
```suggestion
	​   \sum_{i=1}^{n}(x[i] * y[i])
```
看是不是改成单斜杠才是最优的（make html 测试下看）。因为这里是 `r` 字符串，不需要转义。